integration/src/main/java/scripts/testscripts/WorkspacePermissionOperations.java

```diff
@@ -0,0 +1,173 @@
+package scripts.testscripts;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import bio.terra.catalog.api.DatasetsApi;
+import bio.terra.catalog.client.ApiException;
+import bio.terra.catalog.model.CreateDatasetRequest;
+import bio.terra.catalog.model.StorageSystem;
+import bio.terra.rawls.model.WorkspaceACLUpdate;
+import bio.terra.rawls.model.WorkspaceAccessLevel;
+import bio.terra.rawls.model.WorkspaceDetails;
+import bio.terra.testrunner.runner.TestScript;
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import com.google.api.client.http.HttpStatusCodes;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import scripts.client.CatalogClient;
+import scripts.client.RawlsClient;
+
+public class WorkspacePermissionOperations extends TestScript {
```

Comment:
Would it be hard to test both storage systems in a single permissions test, in the way that we do for `DatasetOperations`? If they're testing the same cases, the common code could be shared and it would be easier to see that we're testing the same things for both storage systems.

Reply:
I like that quite a bit. I'll see if can move it over